webapp/channels/src/components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -271,7 +271,7 @@ const SidebarChannelMenu = (props: Props) => {
             menuButton={{
                 id: `SidebarChannelMenu-Button-${props.channel.id}`,
                 class: 'SidebarMenu_menuButton',
-                'aria-label': formatMessage({id: 'sidebar_left.sidebar_channel_menu.editChannel', defaultMessage: 'Channel options'}),
+                'aria-label': `${formatMessage({id: 'sidebar_left.sidebar_channel_menu.editChannel', defaultMessage: 'Channel options'})} for ${props.channel.name}`,
```

Comment:
Instead of direct string concatenation, you might as well create a new string defination for the translation
```
intl.formatMessage({
        id: 'sidebar_left.sidebar_channel_menu.editChannel.ariaLabel',
        defaultMessage: 'xxxx {yy}',
    }, {
        yy: channelName,
    })
```

Reply:
@M-ZubairAhmed I updated the PR as recommended and ran `i18n-extract` (although im not sure what that really did). I re-tested the change and I see it's no longer working with the new changes. Is there a step that I may be missing? The changes make sense for the default message, but will this work for all of the translated messages as well?

For example, if we're using the EN-US translation, will it still add the channelName as well with the new string definition?

Reply:
its because you have not provided a unique id for it, it still uses `sidebar_left.sidebar_channel_menu.editChannel` in other place

Reply:
Ah I see, I missed to see the new id in your example with the addition of the `.ariaLabel` at the end. Thanks for the explanation, will make that update!